src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-legacy-new-relic-exporters.mdx

```diff
@@ -36,7 +36,7 @@ If you still need information about the New Relic Exporter, see the [opentelemet
 
 The [OpenTelemetry Collector with New Relic Exporter](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/collector/nr-exporter-docker) example demonstrates what such a setup might look like.
 
-## Endpoint Configuraion for New Relic exporters [#h2-change-endpoints]
+## Endpoint Configuration for New Relic exporters [#h2-change-endpoints]
```

Comment:
Configuration s/b configuration

Reply:
@barbnewrelic , I was so focused on fixing the typo that I missed the capitalization issue! Thanks!